src/runtests.jl

```diff
@@ -1,28 +1,44 @@
-using Test
 using LinearAlgebra
```

Comment:
What is the purpose of this entire file?

Reply:
removed